### DISCUSSION TITLE
Flash Builder: Export Release Build fails with the Harman AIR SDK for Windows

### DISCUSSION THREAD

**Discussion by liviu-raican** [airsdk/Adobe-Runtime-Support#1379] [Q&A] [answered]
Hi,

I have a Flex project for desktop (uses AIR), managed in Flash Builder 4.7 (x64), which I have been trying to update to use the Harman AIR SDK.

I got the SDK for Flex devs from [their download page](https://airsdk.harman.com/download) and followed their instruction sof overlaying the new SDK files on op of an existing Flex SDK. I did that on top of Flex versions 4.6.0, 4.16.0 and 4.16.1, and the project build is successful every time, but whenever I try to "Export Release Build" from FB, it just silently fails without finishing the task. I am using a self-generated certificate to sign the app, which is fine because I know it's not expired.

The log file in the .metadata folder does not say anything about what happened during the export process.

Does anyone have any experience with this, or any ideas? Or is there some other log file I could check out?

Thank you

**Comment by koreaflash** [comment]
Is there no message in the window? [Window > Show View > Error Log]
![error_log](https://user-images.githubusercontent.com/36652089/143149582-b1d19f6e-8cd9-4e89-95f3-2a5b86cca19f.jpg)

**Comment by ajwfrost** [comment]
We've heard reports of this from time to time... I believe one of the issues that seems to cause this is if there is any customisation of the output application name/folder.

A couple of options to help diagnose things - as well as the error log (per the above):
- can you opt to keep the interim build output and see whether that does actually get created
- if you find the AIR SDK folder, in "lib" there is a config file "adt.cfg", please set the `DebugOut=true` setting and then try again: if the process gets beyond compiling and starts the compilation, there should then be a file "adt.log" generated in your user folder (c:\users\username).

Does this happen every time you build, or just if you have already built the software? So e.g. if you turn off automatic builds, shut down Flash Builder and re-start, does it work the first time?

thanks

**Comment by liviu-raican** [comment, marked as answer]
Thank you for your suggestions! I managed to complete an Export Release Build.

I figured out what was causing this silent fail of the Export Release Build: as the Release Notes state, the JRE used should be at least 1.8. I think that FB uses by default a 1.6 JRE, here: **C:\Program Files\Adobe\Adobe Flash Builder 4.7 (64 Bit)\jre**.

I write here my findings so that others may save themselves a few days of research:

- you have to replace the contents of the _jre_ folder (above) with the contents of a 1.8.0 JRE installation. FB has to be closed while doing so.
- In order for your project to work with embedded fonts, and to work with self-generated certificates as it did before, a working jre for me was **jre 1.8.0.112**. The newest jre as of this writing (1.8.0.291) was not working with the above mentioned features.
- just a note: there was never an **adt.log** file created in _C:/Users/[myuser]_. Not sure why. _DebugOut=true_ was uncommented in **adt.cfg**

**Reply by raresn** [reply]
Salut! Si noi suntem din Romania - daca vrei sa discutam de as3, da-mi un semn :) 0743-033-722
